check_yum

```diff
@@ -332,17 +332,20 @@ class YumTester:
 		if number_updates == 0:
 			status = OK
 			message = "0 Updates Available"
+			perfdata = "updates=0"
 		else:
 			if self.no_warn_on_updates:
 				status = OK
 			else:
 				status = CRITICAL
 			if number_updates == 1:
 				message = "1 Update Available"
+				perfdata = "updates=1"
 			else:
 				message = "%s Updates Available" % number_updates
+				perfdata = "updates=%s" % number_updates
```

Comment:
Why not just doing one
perfdata = "updates=%s" % number_updates
after the if-clause?